bootstrap/server.go

```diff
@@ -95,42 +89,39 @@ func Start(configFilePath string) {
 	plugin.SetPluginConfig(&cfg.Plugin)
 
 	// 初始化存储层
-	store.SetStoreConfig(&cfg.Store)
 	var s store.Store
+	store.SetStoreConfig(&cfg.Store)
 	s, err = store.GetStore()
 	if err != nil {
-		fmt.Printf("[ERROR] get store fail: %v", err)
-		return
+		return "", errors.Wrap(err, "get store fail")
 	}
 
 	// 开启进入启动流程，初始化插件，加载数据等
 	var tx store.Transaction
 	tx, err = StartBootstrapInOrder(s, cfg)
 	if err != nil {
 		// 多次尝试加锁失败
-		fmt.Printf("[ERROR] bootstrap fail: %v\n", err)
-		return
+		return "", errors.Wrap(err, "bootstrap fail")
 	}
-	err = StartComponents(ctx, cfg)
-	if err != nil {
-		fmt.Printf("[ERROR] start components fail: %v\n", err)
-		return
+
+	if err = StartComponents(ctx, cfg); err != nil {
+		return "", errors.Wrap(err, "start components fail")
 	}
+
 	errCh := make(chan error, len(cfg.APIServers))
 	servers, err := StartServers(ctx, cfg, errCh)
 	if err != nil {
-		fmt.Printf("[ERROR] start servers fail: %v\n", err)
-		return
+		return "", errors.Wrap(err, "start servers fail")
 	}
 
 	if err := polarisServiceRegister(&cfg.Bootstrap.PolarisService, cfg.APIServers); err != nil {
-		fmt.Printf("[ERROR] register polaris service fail: %v\n", err)
-		return
+		return "", errors.Wrap(err, "register polaris service fail")
 	}
 	_ = FinishBootstrapOrder(tx) // 启动完成，解锁
-	fmt.Println("finish starting server")
 
 	RunMainLoop(servers, errCh)
+
+	return string(config), nil
```

Comment:
It will block here, unless there is an exception or the program actively exits, it will go to this return